clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntity.java

```diff
@@ -39,9 +40,11 @@
 public class EhcacheClientEntity implements Entity {
 
   private final EntityClientEndpoint<EhcacheEntityMessage, EhcacheEntityResponse> endpoint;
+  private final LifeCycleMessageFactory messageFactory;
 
-  public EhcacheClientEntity(EntityClientEndpoint<EhcacheEntityMessage, EhcacheEntityResponse> endpoint) {
+  public EhcacheClientEntity(EntityClientEndpoint<EhcacheEntityMessage, EhcacheEntityResponse> endpoint, LifeCycleMessageFactory messageFactory) {
```

Comment:
I may have missed some discussion about this but what't the point of passing in a `LifeCycleMessageFactory`?  Unless we're going to configure it (which would require some odd interactions with the server in order to work properly), we're only ever going to use one type of this is a client -- the one against which the current version of the client is written.  I'd prefer direct allocation over another level of indirection without a purpose.